wp_cli/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+use std::{ffi::OsStr, process::Command};
+
+mod wp_cli_settings;
+mod wp_cli_users;
+
+pub use wp_cli_settings::*;
+pub use wp_cli_users::*;
+
+fn run_wp_cli_command<I, S>(args: I) -> std::process::Output
+where
+    I: IntoIterator<Item = S>,
+    S: AsRef<OsStr>,
+{
+    let mut c = Command::new("wp");
+    c.arg("--allow-root")
```

Comment:
If you need this, the process is running as the wrong user – I'd strongly suggest modifying this to run as `www-data`